analyses/data-pre-release-qc/results/cnv-gatk-samples-missing-in-histologies.tsv

```diff
@@ -0,0 +1,106 @@
+Kids_First_Biospecimen_ID
```

Comment:
@zzgeng I think these are the WXS samples - can you make sure for gatk we only are checking that WGS are in the GATK file?

Reply:
I think this file showed the sample that are in gatk file but not histologies file. We have a step to select only WGS samples from histologies, which explains why we see these samples in this file.

Reply:
But this is sayin gatk cnv contains wxs samples? It should only contain wgs- can you check? We may need a re-merge.

Reply:
Oh I think I know what happened here. We changed the input and this part of the code before. This file is generated from previous code and since now we do not have any missing sample, this file is never overwritten. I will add a code to remove all files in the results file before each run to prevent this kind of thing happen again.

Reply:
Ahh, ok makes sense, yeah let;s make sure the results here are fresh per release